framework/math/quadratures/spatial/spatial_quadrature.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "framework/math/quadratures/quadrature_order.h"
+#include "framework/mesh/mesh.h"
+#include "framework/object.h"
+#include <vector>
+
+namespace opensn
+{
+
+class SpatialQuadrature : public Object
+{
+protected:
+  bool verbose_;
+
+  explicit SpatialQuadrature(const InputParameters& params)
+    : Object(params),
+      verbose_(params.GetParamValue<bool>("verbose")),
+      order_(static_cast<QuadratureOrder>(params.GetParamValue<int>("order")))
+  {
+  }
+
+  explicit SpatialQuadrature(QuadratureOrder order) : verbose_(false), order_(order) {}
+
+public:
+  QuadratureOrder order_;
+  std::vector<Vector3> qpoints_;
+  std::vector<double> weights_;
```

Comment:
Public member variables should not have trailing `_`

Reply:
Fixed.